wiki/pages/Mode - Sweet Scent.md

```diff
@@ -0,0 +1,24 @@
+🍂 [`pokebot-gen3` Wiki Home](../Readme.md)
+
+# 🔄 Sweet Scent Mode
+
+Sweet Scent will make a wild Pokémon automatically appear
+
+Start the mode while in the overworld, in any patch of grass/water/cave with encounters.
+
```

Comment:
nit: Should mention that a Pokémon with the move `Sweet Scent` learned is in your party